docs/artifacts/the-patch-level-gate.html

```
@@ -0,0 +1,106 @@
+<title>The Patch-Level Gate</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)